charts/lakefs/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: lakefs
 description: A Helm chart for running LakeFS on Kubernetes
 type: application
-version: 1.3.0
-appVersion: 1.26.1
+version: 1.4.0
+appVersion: 1.28.0
```

Comment:
Why did you bump minor instead of patch?

Reply:
Right.
Done.